SDL/SDLMain.cpp

```diff
@@ -894,6 +897,10 @@ int main(int argc, char *argv[]) {
 					input.id = 0;
 					NativeTouch(input);
 				}
+				if (g_Config.bMouseControl) {
```

Comment:
You don't really need to check g_Config.bMouseControl here, doesn't seem harmful to always update the deltas.